### DISCUSSION TITLE
Phantom SpeechStarted message at start of stream.

### DISCUSSION THREAD

**Discussion by GuerrillaCoder** [deepgram/community#661] [General help]
### Which Deepgram product feature are you using?

Deepgram API - STT Streaming

### Details

I have an intermittent problem where the API sends a "SpeechStarted" message at the beginning of the stream when there is definitely no speaking going on.

I am using these messages to cut all audio output so it causes some issues.  

It doesn't happen every time but it happens enough.

It almost seems like sometimes the first packet of data sent with the default background levels triggers it.  I am sending Mulaw 8k.

The audio input is from a Twilio media stream.

EDIT:  After some playing around it just seems like VAD is incredibly sensitive.  I'm doing over telephone call so any background noise or crackle seems to set it off.  I think this needs some sensitivity controls.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
